presto-native-execution/presto_cpp/main/PeriodicMemoryChecker.h

```diff
@@ -12,6 +12,7 @@
  * limitations under the License.
  */
 #pragma once
+#include <folly/Singleton.h>
```

Comment:
Ok, this comes because of how the headers work with the basically empty LinuxMemoryChecker.h.

You should move this to the LinuxMemoryChecker.cpp where it is used.

Reply:
Moved to LinuxMemoryChecker.cpp